### DISCUSSION TITLE
Is there any good tips on keep cache size under control?

### DISCUSSION THREAD

**Discussion by moniesoke** [mastodon/mastodon#22210] [Q&A]
The caches takes up 1GB/day on my mastodon server. I want to reduce the income of cache files.

Is there any settings that controls how much the server caches and what it put in cache?

How to clear cache? I used the command:

cd /home/mastodon/live
RAILS_ENV=production bin/tootctl cache clear

but nothing happens to the cache files. (I am using AWS S3 storage).

**Comment by wiegelmann** [comment]
RAILS_ENV=production bin/tootctl media remove
RAILS_ENV=production bin/tootctl preview_cards remove

You can add --days N default is 7 to keep the last N days of files.

**Reply by moniesoke** [reply]
Thanks a lot! The media remove seem to be helpful. 

When I run it, there is a count down and progress bar, the process always got stuck at some point, Then, I got a error message like 

Error processing 109328220400128983: execution expired

When I run it again, the progress started again with less total items to process. After several times of running the media remove I finally reduced the cache size. So Thank you!

I found it sent out 12k delete requests, which does cost some money. 

Is there a way to reduce the things from being cached? For example, Assuming there is only one user in this server, should I follow less people from other servers or follow less hashtags?

**Comment by wiegelmann** [comment]
Setup a cronjob with --days N < 7 and your storage will be happy :)
